docs/source/release_notes.rst

```diff
@@ -8,6 +8,7 @@ Future Release
     * Enhancements
     * Fixes
         * Updated the conda install commands to specify the channel (:pr:`1917`)
+        * Update test cases to cover __main__.py file (:pr:`1927`)
```

Comment:
I think this entry belongs in the `Testing` section. 

Also, I notice below that the entry for updating the DFS error message ended up in the `Documentation Changes` section, but seems like that should be in the `Changes` section instead. Can you also fix that in this PR?

Reply:
made changes